PWGJE/Tasks/ChJetTriggerQATask.cxx

```diff
@@ -179,55 +182,64 @@ struct ChJetTriggerQATask {
       spectra.fill(HIST("vertexZ"),
                    collision.posZ()); // Inclusive Track Cross TPC Rows
 
-      for (auto& trk : tracks) { // loop over filtered tracks in full TPC volume having pT > 100 MeV
+      for (auto const& track : tracks) { // loop over filtered tracks in full TPC volume having pT > 100 MeV
 
-        auto const& originalTrack = trk.track_as<soa::Join<aod::Tracks, aod::TracksExtra>>();
-        spectra.fill(HIST("globalP_tpcglobalPDiff_withoutcuts"), trk.p(), trk.p() - originalTrack.tpcInnerParam());
+        auto const& originalTrack = track.track_as<soa::Join<aod::Tracks, aod::TracksExtra>>();
```

Comment:
doesn't this break now when you run it? You are not subscribing to this table anymore. You should just remove originalTrack, you dont need it anymore

Reply:
The code compiles, but I cannot perform local tests due to this issue:
'DPL internal error - oldestPossibleOutput decreased from 1 to 0'

Vit reported this issue 2 months ago; I contacted him, and he said he did not receive a response from the developers.

I am not sure that I got your idea correctly. To which table do you point? We use 'originalTrack' variable to get extra track information, for instance, 'tpcInnerParam()' and 'itsNCls()' which are not defined in our jet-track tables.

Reply:
But you process function is not subscribing to this table : soa::Join<aod::Tracks, aod::TracksExtra>
So it will break when you run it

Reply:
you need to add soa::Join<aod::Tracks, aod::TracksExtra> const&   to the end of your porcess function

Reply:
> you need to add soa::Join<aod::Tracks, aod::TracksExtra> const& to the end of your process function

Thanks, fixed with the new commit